moveit_planners/chomp/chomp_interface/package.xml

```diff
@@ -1,28 +1,30 @@
-<package format="2">
+<?xml version="2.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
   <name>moveit_planners_chomp</name>
-  <description>The interface for using CHOMP within MoveIt</description>
-
   <version>1.1.5</version>
+  <description>The interface for using CHOMP within MoveIt</description>
   <author email="gjones@willowgarage.com">Gil Jones</author>
 
   <maintainer email="chitt@live.in">Chittaranjan Srinivas Swaminathan</maintainer>
-  <maintainer email="henningkayser@picknik.ai">Henning Kayser</maintainer>
+  <maintainer email="dave@picknik.ai">Dave Coleman</maintainer>
```

Comment:
I don't think Dave wants to be listed as maintainer here anymore. Please revert

Reply:
Fixed.